src/foundry/foundry.js/audioHelper.d.ts

```diff
@@ -1,105 +1,111 @@
 /**
- * A helper class to provide common functionality for working with HTML5 audio and Howler instances
- * A singleton instance of this class is available as `game.audio`
- *
- * Audio playback in Foundry VTT is managed by Howler.js (https://howlerjs.com/). Several methods and
- * attributes in this API return :class:`Howl` instances. See the Howler documentation for details
- * and example usage of the Howl API.
+ * A helper class to provide common functionality for working with the Web Audio API.
+ * https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API
+ * A singleton instance of this class is available as game#audio.
+ * @see Game#audio
  */
 declare class AudioHelper {
   constructor();
 
   /**
-   * The set of Howl instances which have been created for different audio paths
-   * @defaultValue `{}`
+   * The primary Audio Context used to play client-facing sounds.
+   * The context is undefined until the user's first gesture is observed.
+   * @defaultValue `undefined`
    */
-  sounds: Record<string, AudioHelper.Sound>;
+  context: AudioContext | undefined;
+
+  /**
+   * The set of AudioBuffer objects which are cached for different audio paths
+   */
+  buffers: Map<string, AudioBuffer>;
+
+  /**
+   * The set of singleton Sound instances which are cached for different audio paths
+   */
+  sounds: Map<string, Sound>;
+
+  /**
+   * Get an array of the Sound objects which are currently playing.
+   * @remarks It's not actually an `Array` but a `Map`.
+   */
+  playing: Map<number, Sound>;
 
   /**
    * A user gesture must be registered before audio can be played.
    * This Array contains the Howl instances which are requested for playback prior to a gesture.
    * Once a gesture is observed, we begin playing all elements of this Array.
    * @defaultValue `[]`
+   * @remarks It's not actually Howl instances (Howler is not used anymore) but function that will be executed.
    */
-  pending: any[];
+  pending: (() => void)[];
 
   /**
    * A flag for whether video playback is currently locked by awaiting a user gesture
-   * @defaultValue `false`
+   * @defaultValue `true`
    */
   locked: boolean;
 
   /**
    * Audio Context singleton used for analysing audio levels of each stream
    * Only created if necessary to listen to audio streams.
    * @defaultValue `null`
+   * @internal
```

Comment:
Why internal?

Reply:
@cswendrowski told me we should consider it part of the private API:
https://discord.com/channels/170995199584108546/811676497965613117/873052279589208064